articles/aks/private-clusters.md

```diff
@@ -151,7 +151,7 @@ As mentioned, virtual network peering is one way to access your private cluster.
 
 2. The private DNS zone is linked only to the VNet that the cluster nodes are attached to (3). This means that the private endpoint can only be resolved by hosts in that linked VNet. In scenarios where no custom DNS is configured on the VNet (default), this works without issue as hosts point at 168.63.129.16 for DNS that can resolve records in the private DNS zone because of the link.
 
-3. In scenarios where the VNet containing your cluster has custom DNS settings (4), cluster deployment fails unless the private DNS zone is linked to the VNet that contains the custom DNS resolvers (5). This link can be created manually after the private zone is created during cluster provisioning or via automation upon detection of creation of the zone using event-based deployment mechanisms (for example, Azure Event Grid and Azure Functions).
+3. In scenarios where the VNet containing your cluster has custom DNS settings (4), cluster deployment fails unless the private DNS zone is linked to the VNet that contains the custom DNS resolvers (5). This link can be created manually after the private zone is created during cluster provisioning or via automation upon detection of creation of the zone using event-based deployment mechanisms (for example, Azure Event Grid and Azure Functions). Additionally to avoid the cluster failure on first time deployment the aks cluster can be deployed with the private DNS Zone resource Id. This only works from API version Microsoft.ContainerService/managedClusters@2022-07-01. Using a version older than this in bicep or arm templates leads to an invalid template.
```

Comment:
Suggest rewording to - To avoid cluster failure during initial deployment, the cluster can be deployed with the private DNS Zone resource ID. This only works with resource type Microsoft.ContainerService/managedCluster and API version 2022-07-01. Using an older version with an ARM template or Bicep resource definition is not supported.

Reply:
Agreed. Better gramatically